buildbot/convert_to_vtk.py

```
@@ -269,11 +269,11 @@ def write_dic_to_vtk(dic, filename):
     dic = {}
 
     for fname in file_list:
-        print("converting : {} t = {}".format(fname, tval))
+        print(f"converting : {fname} t = {tval}")
         dic_tmp = get_plot_patchdata(fname)
 
         for k in dic_tmp.keys():
```

Ruff (0.16.0)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)